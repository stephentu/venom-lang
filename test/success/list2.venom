a = [1,2,3];
print (a[0]);
print (a[1]);
print (a[2]);

b = ["", "hi"];
print (b[0]);
print (b[1]);

a[0] = 10;
a[1] = 20;
a[2] = 30;
print (a[0]);
print (a[1]);
print (a[2]);

b[0] = "foo";
b[1] = "bar";
print (b[0]);
print (b[1]);
