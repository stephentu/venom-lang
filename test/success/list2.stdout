1
2
3

hi
10
20
30
foo
bar
